docs/development.md

```diff
@@ -27,6 +28,23 @@ help you get started.
 
 ## Prerequisites
 
+### Binaries
+
+The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
```

Comment:
With the removal of managed test fixture, `etcd` and `kube-apiserver` are no longer required.  `helm` and `golangci-lint` are new dependencies.  I think it would be preferable to skip naming the dependencies and just refer the user to the download script.  This would avoid having to maintain the docs as our dependencies change.

Reply:
`etcd` and `kube-apiserver` is needed by `sync-up-helm-chart.sh`. So, they are required.

I agree to remove binary list here to decrease the maintenance effort.